NUModules/NUModule.j

```diff
@@ -1187,7 +1222,13 @@ NUModuleTabViewModeIcon                = 2;
 
     CPLog.debug("PAGINATION: Loading page #%@ using fetcher %@", aPage, aFetcher);
 
-    var ID = [aFetcher fetchWithMatchingFilter:[self filter]
+    var filter;
+    if ([_categories count])
+        filter = [self _categoryFilterWithFetcher:aFetcher]
```

Comment:
The name of this method is actually confusing.
For now, we have:
```
- (CPPredicate)_categoryFilterWithFetcher:(NURESTFetcher)aFetcher
```
It will not always return a predicate. It can return `nil`, or filter `string`.
I would suggest:
```
- (id)_currentPaginationFilterForFetcher:(NURESTFetcher)aFetcher
```